shared_model/backend/protobuf/transaction_responses/proto_tx_response.hpp

```diff
@@ -9,81 +9,51 @@
 #include <limits>
```

Comment:
Is it possible to define `max_priority` in cpp file, so that `limits` is not included in hpp file?